src/bun.js/bindings/bindings.cpp

```diff
@@ -97,19 +97,19 @@ static void copyToUWS(WebCore::FetchHeaders* headers, UWSResponse* res)
     auto& internalHeaders = headers->internalHeaders();
 
     for (auto& value : internalHeaders.getSetCookieHeaders()) {
-        res->writeHeader(std::string_view("set-cookie", 10), std::string_view(reinterpret_cast<const char*>(value.characters8()), value.length()));
+        res->writeHeader(std::string_view("set-cookie", 10), std::string_view(value.utf8().data(), value.length()));
```

Comment:
I did not dig into how WTFString lifetimes are managed; will the creation of this temporary leak?